apps/cookbook/src/app/showcase/accordion-showcase/accordion-showcase.component.html

```diff
@@ -1,54 +1,71 @@
 <div class="example">
-  <h2>Accordion</h2>
   <p>
     An accordion is a vertically stacked list of headers that reveal or hide related sections of
     content.
   </p>
+  <h2>Setup</h2>
   <p>
-    <em>
-      <strong>Note:</strong>
-      Accordion uses Angular animations for animating between its collapsed and expanded state. As
-      no other components depend on the package it is not bundled with Kirby for performance reasons
-      and should instead be provided in the application as detailed in the Angular docs for
-      <a href="https://angular.dev/guide/animations#enabling-the-animations-module">
-        enabling the animations module
-      </a>
-      .
-    </em>
+    Accordions uses Angular animations for animating between its collapsed and expanded state. As no
+    other components depend on the package it is not bundled with Kirby for performance reasons and
+    should instead be provided in the application as detailed in the Angular docs for
+    <a href="https://angular.dev/guide/animations#enabling-the-animations-module">
+      enabling the animations module
+    </a>
+    .
   </p>
+  <h2>Simple accordion</h2>
   <cookbook-example-viewer [html]="accordionDefaultExample.template">
     <cookbook-accordion-default-example
       #accordionDefaultExample
     ></cookbook-accordion-default-example>
   </cookbook-example-viewer>
 
-  <h2>Accordion with an initially expanded item</h2>
+  <h3>With initially expanded item</h3>
+  <p>
+    It is possible to have an accordion expanded initially to set focus on a specific information.
+  </p>
   <cookbook-example-viewer [html]="accordionExpandedExample.template">
     <cookbook-accordion-expanded-example
       #accordionExpandedExample
     ></cookbook-accordion-expanded-example>
   </cookbook-example-viewer>
 
   <h2>Accordion inside a card</h2>
+  <p>
+    Accordions may be placed within a card without further adjustments needed to make it stand out
+    from the canvas.
```

Comment:
Consider reusing the wording from data table directly, for consistency: 
```suggestion
    The accordion can be wrapped in a Kirby Card to make it stand out from the canvas.
```